keps/prod-readiness/sig-scheduling/1923.yaml

```diff
@@ -0,0 +1,6 @@
+kep-number: 1923
```

Comment:
Why are you adding this file?

Reply:
I take this as a reference, 
https://github.com/kubernetes/enhancements/blob/7e0d64253807d6ccaa7330186dda1da099bea4a3/keps/prod-readiness/sig-api-machinery/1040.yaml#L1-L3

And also the validate script here:
https://github.com/kubernetes/enhancements/blob/7e0d64253807d6ccaa7330186dda1da099bea4a3/cmd/kepval/main_test.go#L108-L110

Reply:
It looks like it's a new thing in 1.21 I wasn't aware of.
Looks good.